lib/PrivilegedDonorContacts/PrivilegedDonorContacts.js

```diff
@@ -0,0 +1,62 @@
+import { noop } from 'lodash';
+import PropTypes from 'prop-types';
+import { useEffect, useState } from 'react';
+import { FieldArray } from 'react-final-form-arrays';
+
+import {
+  Col,
+  Row,
+} from '@folio/stripes/components';
+
+import { defaultColumnMapping } from './constants';
+import { useFetchPrivilegedContacts } from './hooks';
+import { PrivilegedDonorContactsContainer } from './PrivilegedDonorContactsContainer';
+
+export function PrivilegedDonorContacts({ name, privilegedContactIds, onChange, ...rest }) {
+  const [contactIds, setContactIds] = useState(privilegedContactIds);
+  const { contacts, isLoading } = useFetchPrivilegedContacts(contactIds, { keepPreviousData: true });
+
+  useEffect(() => {
+    setContactIds(privilegedContactIds);
+  }, [privilegedContactIds]);
+
+  const onSetContactIds = (values = []) => {
+    setContactIds(values);
+    onChange(values);
+  };
+
+  return (
+    <Row>
+      <Col xs={12}>
+        <FieldArray
+          name={name}
+          id={name}
+          component={PrivilegedDonorContactsContainer}
```

Comment:
The `PrivilegedDonorContactsContainer` component's name is perceived as a smart wrapper for `PrivilegedDonorContacts`, although it is not. Could you please specify another name for this component?